src/utils/validateBurnedAmount/validateAvaxBurnedAmountEtna.test.ts

```diff
@@ -0,0 +1,282 @@
+    },
+    {
+      name: 'create subnet',
+      unsignedTx: newCreateSubnetTx(
+        testContext,
+        [utxoMock],
+        [testAddress1],
+        [testAddress1],
+      ),
+      baseFee: 3430000n,
+      burnedAmount: 3930000n,
+    },
+    {
+      name: 'create blockchain',
+      unsignedTx: newCreateBlockchainTx(
+        testContext,
+        [utxoMock],
+        [testAddress1],
+        'subnet',
+        'chain',
+        'vm',
+        ['fx1', 'fx2'],
+        {},
+        [0],
+      ),
+      baseFee: 5340000n,
+      burnedAmount: 5660000n,
+    },
+    {
+      name: 'add subnet validator',
+      unsignedTx: newAddSubnetValidatorTx(
+        testContext,
+        [utxoMock],
+        [testAddress1],
+        nodeId().toString(),
+        0n,
+        1n,
+        2n,
+        'subnet',
+        [0],
+      ),
+      baseFee: 4660000n,
+      burnedAmount: 4960000n,
+    },
+    {
+      name: 'remove subnet validator',
+      unsignedTx: newRemoveSubnetValidatorTx(
+        testContext,
+        [utxoMock],
+        [testAddress1],
+        nodeId().toString(),
+        Id.fromHex(testSubnetId).toString(),
+        [0],
+      ),
+      baseFee: 4420000n,
+      burnedAmount: 4420000n,
+    },
+    {
+      name: 'add permissionless validator (subnet)',
+      unsignedTx: newAddPermissionlessValidatorTx(
+        testContext,
+        [utxoMock],
+        [testAddress1],
+        nodeId().toString(),
+        Id.fromHex(testSubnetId).toString(),
+        0n,
+        120n,
+        1800000n,
+        [],
+        [],
+        1,
+        {},
+        1,
+        0n,
+        blsPublicKeyBytes(),
+        blsSignatureBytes(),
+      ),
+      baseFee: 6570000n,
+      burnedAmount: 7570000n,
+    },
+    {
+      name: 'add permissionless delegator (subnet)',
+      unsignedTx: newAddPermissionlessDelegatorTx(
+        testContext,
+        [utxoMock],
+        [testAddress1],
+        nodeId().toString(),
+        Id.fromHex(testSubnetId).toString(),
+        0n,
+        120n,
+        1800000n,
+        [],
+        {},
+        1,
+        0n,
+      ),
+      baseFee: 4850000n,
+      burnedAmount: 4900000n,
+    },
+    {
+      name: 'transfer subnet ownership',
+      unsignedTx: newTransferSubnetOwnershipTx(
+        testContext,
+        [utxoMock],
+        [testAddress1],
+        Id.fromHex(testSubnetId).toString(),
+        [0, 2],
+        [testAddress2],
+      ),
+      baseFee: 5300000n,
+      burnedAmount: 5900000n,
+    },
+  ];
+
+  describe.each(testData)('$name', ({ unsignedTx, baseFee, burnedAmount }) => {
+    it('returns true if burned amount is correct', () => {
+      const result = validateAvaxBurnedAmountEtna({
+        unsignedTx,
+        burnedAmount,
+        baseFee,
+        feeTolerance: 20,
+      });
+
+      expect(result).toStrictEqual({
+        isValid: true,
+        txFee: burnedAmount,
+      });
+    });
+
+    it('returns false if burned amount is not correct', () => {
```

Comment:
Do we want a test to validate that this will be false if the fee is too low as well?

Reply:
yes, added to the tests.